ecl/eclagent/eclgraph.cpp

```diff
@@ -591,7 +591,8 @@ bool EclGraphElement::prepare(IAgentContext & agent, const byte * parentExtract,
             break;
         case TAKif:
             {
-                Owned<IHThorArg> helper = createHelper(agent, NULL);
+                assertex(!subgraph->owner);
```

Comment:
Seems an odd construct - assert that it is null before passing it? How is that better than just passing null ?